app/src/main/res/layout/feedback_item.xml

```diff
@@ -1,20 +1,24 @@
-<layout xmlns:android="http://schemas.android.com/apk/res/android">
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
 
   <data>
 
     <variable
       name="htmlContent"
       type="CharSequence" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.player.state.itemviewmodel.FeedbackViewModel" />
   </data>
 
   <FrameLayout
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:layout_marginStart="20dp"
-    android:layout_marginTop="@dimen/divider_margin_top"
-    android:layout_marginEnd="28dp"
-    android:layout_marginBottom="@dimen/divider_margin_bottom"
-    android:background="@drawable/content_blue_background">
+    android:background="@{viewModel.hasConversationView ? @drawable/content_blue_background : @drawable/transparent_background}"
+    app:layoutMarginBottom="@{viewModel.hasConversationView ? @dimen/margin_8 : @dimen/margin_28}"
+    app:layoutMarginStart="@{viewModel.hasConversationView ? @dimen/margin_20 : @dimen/margin_28}"
+    app:layoutMarginTop="@{viewModel.hasConversationView ? @dimen/margin_8 : @dimen/margin_28}">
```

Comment:
In the future, this will get complicated because I will introduce `isSplitView` and use it to determine the correct margins.  
It will be something like:  
```
if (hasConversationView) {
   if (isSplitView) {
      // some value
   } else {
      // some value   
   }
} else {
   if (isSplitView) {
      // some value
   } else {
      // some value
   }
}
```
but in one line. And this will be repeated for each `layoutMarginX`  
Any idea how to avoid this?

Reply:
We can actually, achieve that easily using `DataBinding`, I will try to share one gist/PR soon.